libass/ass_utils.c

```diff
@@ -150,6 +150,16 @@ void *ass_try_realloc_array(void *ptr, size_t nmemb, size_t size)
     }
 }
 
+bool ass_sv_iequal(ASS_StringView str1, ASS_StringView str2)
```

Comment:
I think this two functions is short enough to be inline in `ass_utils.h`.

Reply:
They're in the .c to avoid needing to resolve the #includes between ass_utils.h and ass_string.h, but maybe all the ASS_StringView util functions should just move to ass_string.h?

Reply:
> maybe all the ASS_StringView util functions should just move to ass_string.h?

Good idea, I should've done that when I introduced `ASS_StringView`. But in that case it's better to move all string-related functions from `ass_utils.h` to `ass_string.h`.

Reply:
Moved these to `ass_string.h` as discussed.